package.json

```diff
@@ -106,6 +106,7 @@
     "electron-updater": "^6.1.1",
     "electron-window-state": "^5.0.3",
     "graphql": "^16.8.1",
+    "graphql-request": "^6.1.0",
```

Comment:
As I remember the we're using graphql for getting staking onformation. Why this additional import needed?

Reply:
Now we use library for react, but here we need plain js solution